### PR TITLE
Stringify params in case they are objects

### PR DESCRIPTION
**Pull request by NavyAdmiral**
For [#167346079](https://www.pivotaltracker.com/story/show/167346079)

### CODE REVIEW

File: operator_ui/@types/operator_ui/index.d.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as dbTypes from '../db'
 import { status, adapterTypes } from '../constants'
 
 interface RunResult {
-  data: { result: string | null }
+  data: { result: string | undefined }
```
Comment (NavyAdmiral) [inline operator_ui/@types/operator_ui/index.d.ts]: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

Comment (rupurt) [reply operator_ui/@types/operator_ui/index.d.ts]: `result?`: string

File: operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx [hunk 2]
```diff
@@ -25,19 +25,23 @@ interface IItemProps extends WithStyles<typeof fontStyles> {
   colATitle: string
   colAValue: string
   colBTitle: string
-  colBValue?: string
+  colBValue: string | undefined
```
Comment (rupurt) [inline operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx]: ✂️ these are equivalent

File: operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx [hunk 3]
```diff
@@ -25,19 +25,23 @@ interface IItemProps extends WithStyles<typeof fontStyles> {
...
 }
 
+type Par = object | string | Array<any> | undefined
+type Ret = string | undefined
+const stringify = (par: Par): Ret => typeof par === 'object' || Array.isArray(par) ? JSON.stringify(par) : par
```
Comment (rupurt) [inline operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx]: Can you add unit test coverage for this

Comment (NavyAdmiral) [reply operator_ui/src/components/JobRuns/TaskExpansionPanel.tsx]: replaced with a proper and popular package

File: operator_ui/package.json [hunk 4]
```diff
@@ -50,6 +50,7 @@
     "formik": "^1.0.3",
     "humps": "^2.0.1",
     "isomorphic-unfetch": "^3.0.0",
+    "javascript-stringify": "^2.0.0",
```
Comment (rupurt) [inline operator_ui/package.json]: 👍

**Pull request merged**